src/sinks/prometheus/exporter.rs

```diff
@@ -375,8 +375,14 @@ fn handle(
 ) -> Response<Body> {
     let mut response = Response::new(Body::empty());
 
-    if !check_auth(&req, auth) {
+    if !authorized(&req, auth) {
         *response.status_mut() = StatusCode::UNAUTHORIZED;
+        let supported_authentication_schemas = http::header::HeaderValue::from_str("Basic, Bearer")
```

Comment:
Nit: you could use `from_static` here, which is a `const` function. That would let you pull this up into a constant defined in this file, if you want, but even doing it inline here should result in better performance.

Reply:
Fixed